src/security/credentials.rs

```diff
@@ -65,6 +68,22 @@ pub enum CertificateRequestType {
         GRPC_SSL_REQUEST_AND_REQUIRE_CLIENT_CERTIFICATE_AND_VERIFY as u32,
 }
 
+/// User-provided callback function for reload cert. [`None`] indicates that
+/// no reloading is needed, and [`Some(ServerCredentialsBuilder)`] indicates
+/// that reloading is needed.
+pub trait UserFetcher {
```

Comment:
`UserFetcher` is unclear. How about `ServerCredentialsFetcher`?

Reply:
Fixed